### ISSUE TITLE
Annotating nhoods by continuous covariates 

### ISSUE THREAD

**Issue by emdann** [MarioniLab/miloR#218]
From discussion with Pe'er lab members: to aid exploratory analysis make functions (and show in vignettes) to relate DA logFC to average value for some continuous covariate in nhoods (e.g. pseudotime, latent factor values, gene signature expression).
Possibly simply extending `annotateNhoods`?

**Comment by MikeDMorgan** [comment]
I think we could also have some bespoke plotting functions to aid this for continuous variables, akin to the beeswarm plots we have for discrete variables.

**Comment by sbenjamaporn** [comment]
Can we use beeswarm plots  for ploting continuous variables? For example, the nhoods were enriched/depleted with age?

**Comment by MikeDMorgan** [comment]
Hi @sbenjamaporn. Generally beeswarm plots are for showing the relationship between a categorical variable and a continuous one rather than two continuous variables. In `miloR` the beeswarm plots are specifically for looking at the nhood enrichment (LFC) vs.s a categorical variable such as cell type annotation.

If you want to look at the average age in each nhood, you could compute the mean age for each and plot this against the LFC for each nhood after DA testing. This might be a vaguely useful sanity check, but could be prone to misinterpretation if the age-related shift is relatively subtle.

**Comment by MikeDMorgan** [comment]
Hi @sbenjamaporn. Beeswarm plots show the relationship between a categorical variable and a continuous one, but you are asking about the relationship between two continuous variables, hence, a beeswarm plot is just the wrong type of visualisation.

There is no issue with using continuous variables for DA testing. If you want to evaluate _post hoc_ the pattern of LFC vs. age then you need to first compute an average value of age for each nhood, as I stated previously.

**Comment by MikeDMorgan** [comment]
Yes, that is exactly what the beeswarm plot function in Milo is designed for. Are you asking about how you interpret the LFC? In which case, that is the same as you would for any log fold change - it is the amount the dependent variable (nhood count) changes per unit of the predictor variable (age in your case).

**Comment by sbenjamaporn** [comment]
Yes, thank you so much @MikeDMorgan
